README.md

```diff
@@ -541,6 +542,29 @@ end
 
 [grape-swagger]: https://github.com/ruby-grape/grape-swagger
 
+## Configuration
+
+Grape counts with a module `Grape.configure` for some basic configuration at load time.
+Currently the configurable settings are:
+
+* `param_builder`: Sets the default [Parameter Builder](#parameters), by default: `Grape::Extensions::ActiveSupport::HashWithIndifferentAccess::ParamBuilder`
```

Comment:
Sets the Parameter Builder, defaults to `Grape::Extensions::ActiveSupport::HashWithIndifferentAccess::ParamBuilder`